at_client/test/notification_service_test.dart

```diff
@@ -381,4 +381,26 @@ void main() {
           NotificationStatusEnum.undelivered);
     });
   });
+
+  group('A group of tests to validate notification subscribe method', () {
+    test('A subscribe returns a new stream for a new regex', () async {
```

Comment:
Tests should have explicit assertions / expectations.

Reply:
Modified the tests to assert on the size of the streamListeners map to verify if a new instances is being created or not.